services/QuillLMS/config/database.yml

```diff
@@ -1,17 +1,17 @@
 default: &default
   pool: <%= ENV.fetch("MAX_THREADS") { 5 } %>
   prepared_statements: false
+  adapter: postgresql
   timeout: 5000
 
-development_env: &development_env
-  adapter: postgresql
+development_shared: &development_shared
   encoding: unicode
   database: emp_gr_development
```

Comment:
I think it's a little confusing to have `database: emp_gr_development` set and then overwritten with test_env.  What about moving it to the development group itself on line 25?